scripts/generate-delta-package-list.sh

```diff
@@ -73,7 +73,11 @@ function collect_rpm_packages() {
 function collect_python_packages() {
   file=$1
   echo "Collecting installed python packages"
-  pip list --format json | jq -r '.[].name' | sort >> "$file"
+  if [ "${OS}" == "redhat8" ] ; then
```

Comment:
You could replace "virtualenv" and "pip" with "python3 -m virtualenv" and "python3 -m pip" in general - we shouldn't rely on anything Python 2 related nowadays, not even on CentOS 7.